src/Files.App/Actions/Open/OpenClassicPropertiesAction.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using System.Runtime.InteropServices;
+using Windows.Win32;
+using Windows.Win32.UI.Shell;
+
+namespace Files.App.Actions
+{
+	internal sealed class OpenClassicPropertiesAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context;
+
+		public string Label
+			=> "OpenClassicProperties".GetLocalizedResource();
+
+		public string Description
+			=> "OpenClassicPropertiesDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph
+			=> new(themedIconStyle: "App.ThemedIcons.Properties");
+
+		public HotKey HotKey
+			=> new(Keys.Enter, KeyModifiers.AltShift);
+
+		public bool IsExecutable =>
+			context.PageType is not ContentPageTypes.Home &&
+			!(context.PageType is ContentPageTypes.SearchResults &&
+			!context.HasSelection);
+
+		public OpenClassicPropertiesAction()
+		{
+			context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+			context.PropertyChanged += Context_PropertyChanged;
+		}
+
+		public Task ExecuteAsync(object? parameter = null)
+		{
+			if (context.HasSelection && context.SelectedItems is not null)
+			{
+				foreach (var item in context.SelectedItems)
+					ExecuteShellCommand(item.ItemPath);
```

Comment:
When multiple items are selected, the action should open a single properties window for multiple items, not a properties window for each item.

Reply:
The simplest solution is to invoke the context menu verb, but encountering an issue "The properties for this item are not available".

Reply:
I couldn't get it to work but you're welcome to try.

Reply:
Turns out the issue I mentioned is only affecting the downloads folder. We should find a way to fix this but I don't think it's a blocker.

Reply:
When I try it, it doesn't work with the error in any folder, not just the downloads folder.

Reply:
What happens if you right click the item to load the shell extensions, and then try the action again?

Reply:
Still the error occurs.

Reply:
In that case it might be best for me to revert the last commit

Reply:
Then this should only be available when one item is selected. When I tried to open the Explorer properties window with many items selected, the app crashed.

Reply:
That sounds like a good plan for now until we can add support for opening the combined properties window.